plugin.video.crackle/resources/lib/globals.py

```diff
@@ -0,0 +1,198 @@
+import sys, os
+import xbmc, xbmcplugin, xbmcgui, xbmcaddon
```

Comment:
It's a minor thing, but recommended import order is:
- Standard Lib
- Third-party libs
- Your modules